src/components/info/info.js

```diff
@@ -61,13 +63,19 @@ const arrayToSentence = (arr, {prefix=undefined, suffix=undefined, capatalise=tr
   return ret + " ";
 };
 
-export const createSummary = (mainTreeNumTips, nodes, filters, visibility, visibleStateCounts, branchLengthsToDisplay) => {
+export const createSummary = (mainTreeNumTips, nodes, filters, visibility, visibleStateCounts, branchLengthsToDisplay, t) => {
   const nSelectedSamples = getNumSelectedTips(nodes, visibility);
   const sampledDateRange = getVisibleDateRange(nodes, visibility);
   /* Number of genomes & their date range */
-  let summary = `Showing ${nSelectedSamples} of ${mainTreeNumTips} genomes`;
+  let summary = t(
```

Comment:
`TypeError: "t is not a function"` Which prevents the Download modal opening (it imports this function, but hasn't been updated to pass in `t`)

Reply:
I had t added, but didn't notice that the one path didn't pass branchLengths so I had t, but to the wrong arg index.